### PR TITLE
Added api endpoint for changing the group name

### PR DESCRIPTION
**Pull request by MoritzHahn1337**

### CODE REVIEW

File: datameta/api/openapi.yaml [hunk 1]
```diff
@@ -493,6 +493,38 @@ paths:
...
+        "204":
+          description: OK
+        '401':
+          description: Unauthorized
+        '403':
+          description: Forbidden
+        '400':
+          $ref: '#/components/responses/ValidationError'
+        '500':
+          $ref: '#/components/responses/ValidationError'
```
Comment (lkuchenb) [inline datameta/api/openapi.yaml:526]:
```suggestion
          description: Internal Server Error
```

File: datameta/api/openapi.yaml [hunk 2]
```diff
@@ -736,6 +768,15 @@ components:
       items:
         $ref: "#/components/schemas/SubmissionResponse"
 
+    GroupNameChange:
```
Comment (lkuchenb) [inline datameta/api/openapi.yaml:771]: Let's stick to one naming scheme, maybe `GroupUpdateRequest` in accordance with `FileUpdateRequest`

File: datameta/api/openapi.yaml [hunk 3]
```diff
@@ -736,6 +768,15 @@ components:
...
+      type: object
+      properties:
+        newGroupName:
```
Comment (lkuchenb) [inline datameta/api/openapi.yaml:774]:
```suggestion
        name:
```
Just use the attribute name here

**Review by lkuchenb — CHANGES_REQUESTED** [review]

File: datameta/api/openapi.yaml (hunk 3 above)
Comment (MoritzHahn1337) [reply datameta/api/openapi.yaml]: Done

File: datameta/api/openapi.yaml (hunk 2 above)
Comment (MoritzHahn1337) [reply datameta/api/openapi.yaml]: Done

**Review by lkuchenb — CHANGES_REQUESTED** [review]
Change API version from `0.0.1` to `0.1.0`

**Review by lkuchenb — APPROVED** [review]

**Comment by lkuchenb** [comment]
Thanks!

**Pull request merged**